tests/console/systemtap.pm

```diff
@@ -22,6 +22,16 @@ sub run {
     $self->select_serial_terminal;
     prepare_for_kdump();
     zypper_call("in systemtap systemtap-docs kernel-devel systemtap-server");
+    script_run('ti=$(ls /lib/modules/ | grep $(uname -r) | grep -oP ".*(?=-default)") && zypper se -i -s kernel-default-devel | grep $ti > vertmp');
+    my $vers = script_output('cat vertmp');
+    if (!$vers) {
+        my $dev = zypper_call("se -i -s kernel-default-devel", exitcode => [0, 104]);
+        if ($dev ne '104') {
+            die "Installed kernel-devel does not match kernel version";
+        } else {
+            die "kernel-devel package is required but not installed";
```

Comment:
can you add found and expected to the die message?

Reply:
No, good idea - going to add that now. Thanks!